### PR TITLE
[AI Gateway] OpenAI compat endpoints

### PR DESCRIPTION
**Pull request by kodster28**

### CODE REVIEW

File: content/ai-gateway/_partials/_workersai.md [hunk 1]
```diff
@@ -31,5 +31,29 @@ header: Request to Workers AI text classification model
 curl https://gateway.ai.cloudflare.com/v1/{account_id}/{gateway_slug}/workers-ai/@cf/huggingface/distilbert-sst-2-int8 \
   --header 'Authorization: Bearer {cf_api_token}' \
   --header 'Content-Type: application/json' \
-  --data '{ "text": "This pizza is amazing!" }'
```
Comment (crwaters16) [inline content/ai-gateway/_partials/_workersai.md:34]: 😂

**Review by crwaters16 — APPROVED** [review]

**Comment by kodster28** [comment]
@G4brym, mind taking a quick look to make sure I'm not making anything up?

Tested locally and worked fine for me.

File: content/ai-gateway/_partials/_workersai.md (hunk 1 above)
Comment (kodster28) [reply content/ai-gateway/_partials/_workersai.md:34]: I mean, AI needs to know about pizza too

File: content/ai-gateway/_partials/_workersai.md [hunk 2]
```diff
@@ -31,5 +31,29 @@ header: Request to Workers AI text classification model
...
 ```
+
+## OpenAI compatible endpoints
+
+{{<render file="_openai-compatibility.md" productFolder="workers-ai">}}
+<br/>
+
+```bash
+---
+header: Request to OpenAI compotible endpoint
```
Comment (KianNH) [inline content/ai-gateway/_partials/_workersai.md]:
```suggestion
header: Request to OpenAI compatible endpoint
```

**Pull request merged**